src/SonarScanner.MSBuild.Common/OperatingSystemProvider.cs

```diff
@@ -71,4 +78,10 @@ private static PlatformOS CurrentOperatingSystem()
 #else
         File.Exists("/System/Library/CoreServices/SystemVersion.plist");
 #endif
+
+    // See: https://www.freedesktop.org/software/systemd/man/latest/os-release.html
+    // Examples: "ID=alpine", "ID=fedora", "ID=debian".
+    private bool IsAlpineRelease(string releaseInfoFilePath) =>
+        fileWrapper.Exists(releaseInfoFilePath)
+        && fileWrapper.ReadAllText(releaseInfoFilePath).Contains("ID=alpine");
```

Comment:
`ReadAllText` can throw if permissions are insufficient (Exists returns true in that case). Add logic and UTs for that case.